beekeeper-scheduler-apiary/src/main/resources/db/migration/V2_1__Create_unpartitioned_table_housekeeping_table_and_alter_path_table_name_and_rename_path_status_column.sql

```diff
@@ -0,0 +1,19 @@
+USE beekeeper;
+
+CREATE TABLE IF NOT EXISTS housekeeping_metadata (
+  id BIGINT(20) AUTO_INCREMENT,
+  database_name VARCHAR(512),
+  table_name VARCHAR(512),
+  housekeeping_status VARCHAR(50) NOT NULL,
+  cleanup_delay VARCHAR(50) NOT NULL,
+  creation_timestamp TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  modified_timestamp TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  cleanup_timestamp TIMESTAMP NOT NULL,
+  cleanup_attempts INT NOT NULL DEFAULT 0,
+  client_id VARCHAR(512),
+  lifecycle_type varchar(255) NOT NULL,
+  PRIMARY KEY (id)
+);
+
+ALTER TABLE path RENAME TO housekeeping_path;
```

Comment:
I think that possibly it will be better to have these commands in separate files? What do you think? Just thinking that if the create table succeeds, then another fails, we'll have to roll back the entire thing.

Reply:
True but its all part of 1 migration so I don't know know. If we split it out then do you want the alter commands in different files as well?

Reply:
Yeah i can't find any clear docs on flyway, its probably fine